std/algorithm/comparison.d

```diff
@@ -1194,7 +1194,15 @@ levenshteinDistanceAndPath(alias equals = "a == b", Range1, Range2)
 
 // max
 /**
-Returns the maximum of the passed-in values.
+This function iterates the passed arguments and returns the maximum value.
+
+Params:
+    args = The values to select the maximum from. At least two arguments must
+    be passed.
+
+Returns: 
+    The maximum of the passed-in args. The type of the returned value is a
+    type that is able to store all values of the passed arguments $(D args).
```

Comment:
Well, it's the largest of the argument types, but no larger, so it's the smallest that can fit all of them. :-) Perhaps you can think of a better wording for this, though.